_includes/welcome.html

```diff
@@ -3,6 +3,12 @@ <h2>Welcome to Vespa</h2>
 
 <p class="intro">Serve big data with ease - dynamic responses at any <br> scale in milliseconds with any traffic volume.</p>
 
+
+<p>
+    Vespa is an engine for low-latency computation over large data sets. 
+    It stores and indexes your data such that queries, selection and processing over the data can be performed at serving time.
+</p>
+
```

Comment:
Can you remove this change from this PR? The text (and layout) need some tweaking.